Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "suckit"
-version = "0.1.2"
-edition = "2018"
+version = "0.1.3"
```

Comment:
So this release is "just" a patch? No breaking changes?

Reply:
There is some breaking changes, my bad
Fixed by 898e31a